max_footprint_cost: 128
Inscribed_radius: 0.3
Circumscribed_radius: 0.3
